src/Actions/ManagesWorkflow.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Novu\SDK\Actions;
+
+use Novu\SDK\Resources\Workflow;
+
+trait ManagesWorkflow
+{
+    public function getWorkflows(int $page = 1, int $limit = 10)
+    {
+
+        $response =   $this->get('workflows'. '?' . http_build_query(['page' => $page, 'limit' => $limit]))['data'];
+        return new Workflow($response, $this);
+    }
+
+    public function createWorkflow(array $data)
+    {
+        $response =   $this->post('workflows', $data)['data'];
+        return new Workflow($response, $this);
```

Comment:
There should be a new line before the return. Please format properly. You can look at how it is done in the other code in this repo.